### PR TITLE
[Snyk + GH Actions] Update requirements

### PR DESCRIPTION
**Pull request by nickumia-reisys**
Update requirements
- Updated `requirements.in` + `requirements.txt`
- Auto-generated by [snyk.yml][1]

[1]: https://github.com/gsa/inventory-app/.github/workflows/snyk.yml

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -4,26 +4,26 @@ attrs==22.2.0
...
 ckanext-googleanalyticsbasic==0.2.0
 ckanext-s3filestore @ git+https://github.com/keitaroinc/ckanext-s3filestore.git@8e815a47cd902884ff93fcff96d90349fb6c5459
 ckanext-saml2auth @ git+https://github.com/keitaroinc/ckanext-saml2auth.git@b2c6cfc2f9d6179601017f2d6530cf4ee5331f00
 ckanext-usmetadata==0.2.5
--e git+https://github.com/ckan/ckanext-xloader.git@9db532439dbb8b88f7c9d11f1e573ff6c164e3b9#egg=ckanext_xloader
+-e git+https://github.com/ckan/ckanext-xloader.git@c25e6663a65bacf9693d0de5b23e8492b1ba55df#egg=ckanext_xloader
 ckantoolkit==0.0.7
 click==7.1.2
-cryptography==39.0.1
+cryptography==40.0.1
```
Comment (nickumia-reisys) [inline requirements.txt:23]: Seems questionable...

File: requirements.txt [hunk 2]
```diff
@@ -4,26 +4,26 @@ attrs==22.2.0
...
+botocore==1.29.101
 certifi==2022.12.7
 cffi==1.15.1
 chardet==3.0.4
 ckan==2.9.8
-ckanext-datajson==0.1.15
+ckanext-datajson==0.1.17
 ckanext-dcat-usmetadata==0.3.8
-ckanext-envvars==0.0.2
+ckanext-envvars==0.0.3
```
Comment (nickumia-reisys) [inline requirements.txt:15]: What? How? ... uhh, oh...😌 just CKAN 2.10 support: https://github.com/okfn/ckanext-envvars/releases/tag/v0.0.3

File: requirements.txt [hunk 3]
```diff
@@ -91,14 +91,14 @@ tzlocal==1.3
 unicodecsv==0.14.1
 Unidecode==1.0.22
 urllib3==1.26.5
-watchdog==2.2.1
+watchdog==3.0.0
```
Comment (nickumia-reisys) [inline requirements.txt:94]: Also seems suspicious..

Comment (jbrown-xentity) [reply requirements.txt:94]: Seems ok to me: https://github.com/gorakhargosh/watchdog/blob/master/changelog.rst

**Review by robert-bryson — APPROVED** [review]

**Comment by nickumia-reisys** [comment]
This is defunct as well,
- https://github.com/GSA/data.gov/issues/4217
- #585

**Pull request closed without merging**